web/client/plugins/TOC.jsx

```diff
@@ -117,7 +117,8 @@ const SmartQueryForm = connect((state) => {
             onQuery: createQuery,
             onReset: reset,
             onChangeDrawingStatus: changeDrawingStatus,
-            onToggleDrawer: toggleControl.bind(null, 'drawer', null)
+            closeDrawer: setControlProperty.bind(null, 'drawer', 'enabled', false),
```

Comment:
for the reasons explained above of course these functions should not be passed here.